x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -80,6 +80,7 @@ const ScrollableFlexItem = styled(EuiFlexItem)`
 
 export interface TGridStandaloneProps {
   appId: string;
+  appIdUI: string;
```

Comment:
wouldn't make more sense if we call this prop `casesOwner`. The distinction of _app id_ and _app UI id_ is very specific to security solutions plugin, and it does not give much context of what is it used for.

Reply:
Yes, you are right!